r/man/array.Rd

```diff
@@ -41,14 +41,12 @@ but not limited to strings only)
 }
 
 \section{Usage}{
-
-
-\if{html}{\out{<div class="sourceCode">}}\preformatted{a <- Array$create(x)
+\preformatted{a <- Array$create(x)
```

Comment:
This isn't intentional. I just ran `make doc` locally. Is something about my R out of date, or does that command need an update?